CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
+
+## How can I contribute?
+
+### Submitting an issue
+
+If you have discovered a bug or have a new feature to suggest, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
```

Comment:
```suggestion
If you have discovered a bug, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
```